bot/exts/ping.py

```diff
@@ -0,0 +1,30 @@
+import logging
+
+from disnake import Embed
+from disnake.ext import commands
+
+
+logger = logging.getLogger(__name__)
+
+
+class Ping(commands.Cog):
+    """Send an embed about the bot's ping."""
+
+    def __init__(self, bot: commands.Bot):
```

Comment:
This is more of a nitpick, but it's probably better to typehint to `SirRobin` here and in `setup`.

Reply:
[a8fc686](https://github.com/python-discord/sir-robin/pull/10/commits/a8fc686e88045f580c583045557690e7611d7e08)